docs/docs/command-line-interface.mdx

```diff
@@ -17,11 +17,12 @@ abstract: The command line interface (CLI) gives you easy-to-remember commands f
 |`rasa shell`            |Loads your trained model and lets you talk to your assistant on the command line.                                                         |
 |`rasa run`              |Starts a server with your trained model.                                                                                                  |
 |`rasa run actions`      |Starts an action server using the Rasa SDK.                                                                                               |
-|`rasa visualize`        |Generates a visual representation of your stories.                                                                                                                                                            |
+|`rasa visualize`        |Generates a visual representation of your stories.                                                                                        |
 |`rasa test`             |Tests a trained Rasa model on any files starting with `test_`.                                                                            |
 |`rasa data split nlu`   |Performs a 80/20 split of your NLU training data.                                                                                         |
 |`rasa data convert`     |Converts training data between different formats.                                                                                         |
-|`rasa data validate`    |Checks the domain, NLU and conversation data for inconsistencies.                                                                                             |
+|`rasa data validate`    |Checks the domain, NLU and conversation data for inconsistencies.                                                                         |
+|`rasa data suggest nlu` |Performs data augmentation by paraphrasing and suggests utterances that can be added to the original training data.                       |
```

Comment:
```suggestion
|`rasa data suggest nlu` |Performs data augmentation by suggesting utterances that can be added to the original training data from the augmented dataset.                      |
```

Reply:
Better to make the change directly in the IDE rather than accepting the suggestion here because of the formatting of the table.